### PR TITLE
Registral scrapper optimization

### PR DESCRIPTION
**Pull request by capyvara**
Use the newer full year query instead of doing day-by-day, runs in seconds now, so I've also disabled the caching.

I've checked the data and it's ok.

Also I changed the sort to be "state, date" instead of "date, state", not sure if it breaks something.

Finally I'm including the whole year in the query as the 2019 data may be useful.

### CODE REVIEW

File: obitos_convert.py [hunk 1]
```diff
@@ -27,20 +28,40 @@ def convert_file(filename):
         )[1]
         row["epidemiological_week_2020"] = brazilian_epidemiological_week(date)[1]
 
+        # There are some missing days on the registral, so default to zero
```
Comment (turicas) [inline obitos_convert.py:31]: If the number is not available, the value should be `None` instead.

File: obitos_convert.py [hunk 2]
```diff
@@ -27,20 +28,40 @@ def convert_file(filename):
...
+                row["new_deaths_indeterminate_2020"] = item.indeterminate
+                row["new_deaths_others_2020"] = item.others
+                row["new_deaths_covid19"] = item.covid
+            elif year == 2019:
+                row["new_deaths_sars_2019"] = item.sars
+                row["new_deaths_pneumonia_2019"] = item.pneumonia
+                row["new_deaths_respiratory_failure_2019"] = item.respiratory_failure
+                row["new_deaths_septicemia_2019"] = item.septicemia
+                row["new_deaths_indeterminate_2019"] = item.indeterminate
+                row["new_deaths_others_2019"] = item.others
```
Comment (turicas) [inline obitos_convert.py:64]: I'd refactor this piece with:

```python
# outside function
from obitos_spider import DeathsSpider

DEATH_CAUSES = list(DeathsSpider.causes_map.keys())

        # [...] inside function
        for item in data:
            year = item.date.year
            for cause in DEATH_CAUSES:
                row[f"new_deaths_{cause}_{year}"] = getattr(item, cause)
```

So we maintain coherence with values expected from the scraper (and delete some lines of code! :)

File: obitos_spider.py [hunk 3]
```diff
@@ -61,56 +62,26 @@ def make_registral_request(
...
-                else:
-                    self.add_causes(row, chart_data["2019"], "2019")
-
-                self.add_causes(row, chart_data["2020"], "2020")
-                row["covid"] = chart_data["2020"]["COVID"]
 
-        yield row
+        for br_date, chart in data["chart"].items():
+            day, month, year = br_date.split("/")
+            date = datetime.date(int(year), int(month), int(day))
```
Comment (turicas) [inline obitos_spider.py:81]: Just a note for future refactoring I can do: replace with `date = PtBrDateField.deserialize(br_date)` (`PtBrDateField` is defined inside `web.spiders.spider_es` -- we need first to refactor the spiders, cc @berinhard).

File: obitos_convert.py (hunk 2 above)
Comment (capyvara) [reply obitos_convert.py:64]: One problem is that covid19 doesn't have the year, should I keep it outside?

File: obitos_convert.py (hunk 1 above)
Comment (capyvara) [reply obitos_convert.py:31]: This is ok so? empty fields?

```
2020-01-01,AC,1,1,0,3,2,1,0,4,0,2,1,1,1,7,,,3,2,1,,4,,2,1,1,1,7,,10,12,10,12
2020-01-02,AC,1,1,0,0,0,1,0,6,0,2,0,2,0,6,,,,,2,,10,,4,,3,,13,,7,10,17,22
2020-01-03,AC,1,1,0,0,0,2,0,6,0,3,0,2,0,8,,,,,4,,16,,7,,5,,21,,8,13,25,35
2020-01-04,AC,1,1,0,3,0,1,0,1,0,0,1,1,0,4,,,6,,5,,17,,,2,6,,25,,5,6,30,41
2020-01-05,AC,1,2,0,1,0,0,0,5,0,4,1,0,0,4,,,7,,,,22,,11,3,,,29,,6,9,36,50
```

File: obitos_convert.py (hunk 2 above)
Comment (capyvara) [reply obitos_convert.py]: Nice, I was keeping the code minimal change, but I've refactored this logic to make it less hardcoded.

**Comment by capyvara** [comment]
Build is breaking but seems to be related out of other change

**Pull request merged**